CHANGELOG.ru.md

```diff
@@ -1,5 +1,26 @@
 # История изменений
 
+## 2.3.0
+
+### Крупные изменения
+* Библиотека `bem-core` была обновлена до версии [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
```

Comment:
I think bem.info is much better way to read library documentation so why shouldn't we force users to it?